app/main/views/marketplace.py

```diff
@@ -75,14 +75,22 @@ def terms_and_conditions():
 def get_brief_by_id(framework_framework, brief_id):
     briefs = data_api_client.get_brief(brief_id)
     brief = briefs.get('briefs')
+    brief_responses = (data_api_client.find_brief_responses(brief_id=brief_id)).get("brief_responses")
+    if brief_responses is None:
+        brief_responses = []
+    started_brief_responses_len = len([response for response in brief_responses if "submitted_at" not in response])
+    completed_brief_responses_len = len([response for response in brief_responses if "submitted_at" in response])
+
```

Comment:
It's not overly going to impact performance, but yeah, we could just do the loop once and then deduce `completed_brief_responses_len = len(brief_responses) - started_brief_responses_len`